e2e/ansible/playbooks/resiliency/test-node-eviction-out-of-disk/test-node-eviction-out-of-disk.yml

```diff
@@ -72,7 +72,7 @@
          args:
            executable: /bin/bash
          register: sc_out
-         until: "'created' in sc_out.stdout"
+         until: "'configured' in sc_out.stdout"
```

Comment:
This change is not required, as the test deploys sc every-time and preforms cleanup post test. Can you revert to 'created' only.

Reply:
@dargasudarshan When we ran this playbook in GKE. we have seen "configured" string in stdout so we modified it to configured.
@shashank855 lets include both the strings in until condition. Ex. configured or created in stdout.